packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -108,10 +108,7 @@ abstract class BuildFrameworkCommand extends BuildSubCommand {
   @override
   bool get reportNullSafety => false;
 
-  @protected
-  late final FlutterProject project = FlutterProject.current();
-
-  Future<List<BuildInfo>> getBuildInfos() async {
+  Future<List<BuildInfo>> getBuildInfos(FlutterProject project) async {
```

Comment:
The `project` parameter appears unused. Could this be removed?

Reply:
Done!